README.md

```diff
@@ -50,10 +50,10 @@ Created by [Erik Trautman](http://www.github.com/eriktrautman)
 **Our main focus** is the cultivation and sequencing of the best free resources around the internet. We believe wholeheartedly that everything one needs to know to become employed can be found for free online. However, for the average new learner, the vast quantity \(and widely varying quality\) of resources makes it difficult to make meaningful progress.  Therefore, we have devised three basic steps to fix this.
 
 1. We have created a curriculum that attempts to find the optimal sequencing of information. This curriculum is intended to take someone from knowing literally nothing about web development to a point where they could be employed as a Junior Developer.
-2. For each topic in the curriculum, we try to find the best free resources on the internet that teach that topic.  We will often link multiple resources so as to not rely too much on a single source of information.  If good resources cannot be found, we write our own, but we definitely prefer primary links to external sites.
+2. For each topic in the curriculum, we try to find the best free resources on the internet that teach that topic.  We will often link multiple resources to not rely too much on a single source of information.  If good resources cannot be found, we write our own, but we prefer primary links to external sites.
```

Comment:
I personally prefer to keep the `so as` here. Alternatively, perhaps changing it to `so that we don't rely...`. I like having something extra in there to help smooth the transition and lessen confusion as you reach the later part of the sentence.

Good with deleting the `definitely` here.